contrib/vcloud/benchmarkclient_executor.py

```diff
@@ -108,8 +110,8 @@ def execute_benchmark(benchmark, output_handler):
             cmdLine.extend(["--try-less-memory", str(benchmark.config.tryLessMemory)])
         if benchmark.config.debug:
             cmdLine.extend(["--print-new-files", "true"])
-
-        start_time = benchexec.util.read_local_time()
```

Comment:
This looks like a mistake

Reply:
Exactly. And it tells us the PR was not tested Fixed by 8b0fbfcf3b6ab4c0f91faa90c071f598fd7979e3.